app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -1,65 +1,50 @@
 <template>
-  <div>
+  <div v-if="listDatasets.length">
     <div
-      v-for="({ id, attributes: { slug, name, columns }}, index) in listDatasets"
-      :key="id"
+      v-for="({ attributes: { slug, name }}) in listDatasets"
+      :key="slug"
       class="gobierto-data-sidebar-datasets"
     >
       <div class="gobierto-data-sidebar-datasets-links-container">
         <i
-          :class="{'rotate-caret': toggle !== index }"
```

Comment:
Instead of using the index I prefered to use the slug, because the index when the array of datasets is reordered was confusing to manage, while the slug is inmutable